src/Database/Query.php

```diff
@@ -623,7 +623,7 @@ protected function execute(string|array $sql, array $params = [])
 	/**
 	 * Selects only one row from a table
 	 */
-	public function first(): object|array|false
+	public function first(): mixed
```

Comment:
@adamkiss what else do we need to support here? with what use case do the more specific typings fail?

Reply:
@distantnative anything that your `fetch(Closure)` returns. If it returns a string… `first()` returns a string as well.